app/static/js/app.js

```diff
@@ -74,18 +74,19 @@ function displayPoweringDownUI(restart) {
     "error-panel",
     "remote-screen",
     "keystroke-history",
-    "shutdown-dialog",
   ]) {
     hideElementById(elementId);
   }
-  const shutdownMessage = document.createElement("h2");
+  const shadowroot = document.getElementById("shutdown-wait").shadowRoot;
```

Comment:
Reaching into an element's shadow and poking around breaks the abstraction a little bit. I proposed a rewrite where `shutdown-wait` exposes a `message` property instead.

https://github.com/djclueless/tinypilot/pull/1